src/test/java/com/google/devtools/build/lib/skyframe/rewinding/RewindingTest.java

```diff
@@ -78,6 +78,7 @@ protected void setupOptions() throws Exception {
         "--rewind_lost_inputs",
         "--features=cc_include_scanning",
         "--experimental_remote_include_extraction_size_threshold=0",
+        "--experimental_remote_cache_eviction_retries=0",
```

Comment:
Does this have an effect only after https://github.com/bazelbuild/bazel/pull/25396? I don't know how stacked PRs work.

Reply:
It's relevant already with this PR, which is the first in the stack. Without it, the `lostInputWithRewindingDisabled` test doesn't see the expected exit code. Since build rewinding and action rewinding are somewhat competing strategies for handling lost inputs now, I found it safer to disable build rewinding for the entire test suite.

Reply:
I see now. We're no longer guarding the full build retry on a specific exit code which is only possible in bazel. So now `--experimental_remote_cache_eviction_retries` is load bearing for blaze, when there's a lost input with `--norewind_lost_inputs`. How can we ensure that `--experimental_remote_cache_eviction_retries` continues to be a no-op for blaze?

Reply:
I don't know how these Bazel/Blaze differences are usually handled. If setting the flag to 0 in the global `.blazerc` is not sufficient, maybe the flag default could be changed with copybara?

@coeuvre

Reply:
I can use blazerc if necessary, I was just hoping there was a way to continue keeping the flag completely irrelevant.

Reply:
I think that's the most natural solution and the one that's been used in the past when something like this came up. The only alternative I see would be to add a bit to the lost input exception types that all Bazel call sites set differently, but that feels worse to me.

Reply:
Oh no, another issue. We need to modify invocation policy to force the flag to 0 instead of ignoring it. Otherwise the blazerc value will be ignored. I'll update here when that lands. If you don't hear from me within a week, please ping.